cypress/e2e/issue-list.cy.ts

```diff
@@ -45,10 +45,22 @@ describe("Issue List", () => {
           cy.wrap($el).contains(issue.name);
           cy.wrap($el).contains(issue.message);
           cy.wrap($el).contains(issue.numEvents);
+          cy.wrap($el).contains(issue.numUsers);
           cy.wrap($el).contains(firstLineOfStackTrace);
         });
     });
 
+    it("renders correct event and user data", () => {
+      cy.get("main")
+        .find("tbody")
+        .find("tr")
+        .each(($el, index) => {
+          const issue = mockIssues1.items[index];
+          cy.wrap($el).find("td").eq(-2).should("have.text", issue.numEvents);
+          cy.wrap($el).find("td").eq(-1).should("have.text", issue.numUsers);
```

Comment:
`-2` and `-1` references the last two elements within an array (in this case `Events` and `Users` in `td`).